roles/common/files/sensu_plugins/check-nbd-nonunique.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Checks if same file is opened by multiple /dev/nbdX devices
+
+while getopts 'w:c:h' OPT; do
+  case $OPT in
+    c)  CRIT=$OPTARG;;
+    h)  hlp="yes";;
+    *)  unknown="yes";;
+  esac
+done
+
+# usage
+HELP="
+    usage: $0 [ -w value -c value -p -h ]
+
+        -c --> Critical < value
+        -h --> print this help screen
+"
+
+if [ "$hlp" = "yes" ]; then
+  echo "$HELP"
+  exit 0
+fi
+
+CRIT=${CRIT:=1}
+
+NO_NBD=`ps x | grep qemu-nbd | grep -v grep | awk '{ print $(NF) }' | uniq -d | wc -l`
```

Comment:
You can use `pgrep -f`  instead of `ps x | grep -v grep`.

`grep ... | awk ...` is an unnecessary use of grep.